manifest.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
```

Comment:
If the field is set to null we can probably delete it since null is the default value anyway.